### ISSUE TITLE
No se cumple el Principio de Abierto/Cerrado

### ISSUE THREAD

**Issue by raul40** [raul40/Eventos#3]
https://github.com/raul40/Eventos/blob/6195bdfa7e83448598c436cabba44794e1b7b77e/ConsoleApp1/Program.cs#L17

Este principio no se cumple por que ese método que deberia ser una clase esta abierto a modificaciones.

**Comment by JaimeYah** [comment]
Sí, pero no es la respuesta que se esperaba. Debiste indicar cómo es que se viola y cómo se puede solucionar.
